client/cmdlf.c

```diff
@@ -1044,6 +1045,7 @@ static command_t CommandTable[] =
 	{"awid",        CmdLFAWID,          1, "{ AWID RFIDs...    }"},
 	{"cotag",       CmdLFCOTAG,         1, "{ COTAG RFIDs...   }"},
 	{"em",          CmdLFEM4X,          1, "{ EM4X RFIDs...    }"},
+	{"fdx",         CmdLFFdx,           1, "{ FDX-B tags...    }"},
 	{"gproxii",     CmdLF_G_Prox_II,    1, "{ G Prox II tags...    }"},
```

Comment:
i plan on changing them all later.  demod for formats will be called tags and ones for transponders/chips will be called chips to distinguish between the two.